router/user.go

```diff
@@ -82,11 +82,27 @@ func (h Handlers) GetMe(c echo.Context) error {
 		h.Logger.Error("failed to get session", zap.Error(err))
 		return echo.NewHTTPError(http.StatusInternalServerError, err)
 	}
-	user, ok := sess.Values[sessionUserKey].(User)
+	user_in_session, ok := sess.Values[sessionUserKey].(User)
```

Comment:
めちゃ横槍で申し訳ないけど、プライベートな変数はローワーキャメルケースが標準な気がします
```suggestion
	userInSession, ok := sess.Values[sessionUserKey].(User)
```

Reply:
:koreni_natteru: